tests/snapshots/run_file__override_list_ctr.hvm.snap

```diff
@@ -2,4 +2,6 @@
 source: tests/golden_tests.rs
 input_file: tests/golden_tests/run_file/override_list_ctr.hvm
 ---
-At line 2: LNil is a built-in constructor and should not be overridden.
+At tests/golden_tests/run_file/override_list_ctr.hvm/:1:1: LNil is a built-in constructor and should not be overridden.
```

Comment:
It's inserting an extra `/` at the end of the path.

Reply:
tests/golden_tests/run_file/override_list_ctr.hvm/
vs
tests/golden_tests/run_file/override_list_ctr.hvm